easy_phi/app.py

```diff
@@ -123,16 +157,26 @@ def get(self):
         return response
 
 # URL schemas to RequestHandler classes mapping
-APPLICATION = tornado.web.Application([
-    (r"/", VersionHandler),  # Default page
-    (r"/version", VersionHandler),
-    (r"/modules", ModulesListHandler),
-    (r"/pick_module", SelectModuleHandler),
-    (r"/send_command", SCPICommandHandler),
-    (r"/admin", AdminConsoleHandler)
-])
+application = tornado.web.Application([
+    (r"/", tornado.web.RedirectHandler,
+        {"url": '/static/index.html'}),
+    (r"/api/v1/info", PlatformInfoHandler),
+    (r"/api/v1/modules", ModulesListHandler),
+    (r"/api/v1/module", SelectModuleHandler),
+    (r"/api/v1/scpi", SCPICommandHandler),
```

Comment:
Module handler can still do both GET and POST at the same time, not an issue. 
GET - to select module
POST - to send SCPI command to the module

Please see example
 http://stackoverflow.com/questions/24899201/is-it-possible-to-use-the-same-method-for-both-get-and-post-with-webapp2

Let's see what @user2589 thinks